lua/autorun/client/cl_cfc_optional_addons_core.lua

```diff
@@ -17,11 +14,13 @@ local function mountAddon( id )
         end
 
         for _, filename in pairs( files ) do
-            if string.match(filename, "^lua/autorun/client/.*%.lua") then
+            local isAutorun = string.match( filename, "^lua/autorun/.*%.lua" )
+            local isServer = string.match( filename, "^lua/autorun/server.*%.lua" )
```

Comment:
You could just check if the string starts with lua/autorun/server. Haven't benchmarked it but that might be more efficient.